codex-rs/tui/src/lib.rs

```diff
@@ -277,3 +322,33 @@ fn should_show_login_screen(config: &Config) -> bool {
         false
     }
 }
+
+fn should_show_trust_screen(
+    config: &mut Config,
+    config_toml: &ConfigToml,
+    approval_policy_overide: Option<AskForApproval>,
+    sandbox_mode_override: Option<SandboxMode>,
+    cwd: Option<PathBuf>,
```

Comment:
Why does this take `cwd` as an arg instead of using `config.cwd`?

We should be sure we are honoring `--cwd` if the user passed it in...